### ISSUE TITLE
Exception from 3.0.1

### ISSUE THREAD

**Issue by ZebraFlesh** [cascornelissen/svg-spritemap-webpack-plugin#58]
After upgrading to 3.0.1 (from 3.0.0#master), I'm seeing the following error. Commenting out the svg-spritemap-webpack-plugin usage eliminates it, so it looks like it's the new version. :( The following stack trace backs that up.

node: 10.13.0
webpack: 4.27.1
svg-spritemap-webpack-plugin: 3.0.1
os: windows 10

```
(node:84124) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'providedExports' of undefined
    at fnModule (C:\Users\NOPE\myproject\dls\node_modules\webpack\lib\Stats.js:586:58)
    at Array.map (<anonymous>)
    at obj.chunks.compilation.chunks.map.chunk (C:\Users\NOPE\myproject\node_modules\webpack\lib\Stats.js:658:8)
    at Array.map (<anonymous>)
    at Stats.toJson (C:\Users\NOPE\myproject\node_modules\webpack\lib\Stats.js:616:36)
    at StatsWriterPlugin.emitStats (C:\Users\NOPE\myproject\node_modules\webpack-stats-plugin\lib\stats-writer-plugin.js:72:40)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\NOPE\myproject\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:8:17)
    at AsyncSeriesHook.lazyCompileHook (C:\Users\NOPE\myproject\node_modules\tapable\lib\Hook.js:154:20)
    at Compiler.emitAssets (C:\Users\NOPE\myproject\node_modules\webpack\lib\Compiler.js:363:19)
    at onCompiled (C:\Users\NOPE\myproject\node_modules\webpack\lib\Compiler.js:231:9)
    at hooks.afterCompile.callAsync.err (C:\Users\NOPE\myproject\node_modules\webpack\lib\Compiler.js:552:14)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\NOPE\myproject\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:33:1)
    at AsyncSeriesHook.lazyCompileHook (C:\Users\NOPE\myproject\node_modules\tapable\lib\Hook.js:154:20)
    at compilation.seal.err (C:\Users\NOPE\myproject\node_modules\webpack\lib\Compiler.js:549:30)
    at AsyncSeriesHook.eval [as callAsync] (eval at create (C:\Users\NOPE\myproject\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:6:1)
    at AsyncSeriesHook.lazyCompileHook (C:\Users\NOPE\myproject\node_modules\tapable\lib\Hook.js:154:20)
    at hooks.optimizeAssets.callAsync.err (C:\Users\NOPE\myproject\node_modules\webpack\lib\Compilation.js:1323:35)
    at _err3 (eval at create (C:\Users\NOPE\myproject\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:26:1)
    at compilation.hooks.optimizeAssets.tapAsync (C:\Users\NOPE\myproject\node_modules\svg-spritemap-webpack-plugin\lib\index.js:245:32)
    at _err2 (eval at create (C:\Users\NOPE\myproject\node_modules\tapable\lib\HookCodeFactory.js:32:10), <anonymous>:22:1)
    at SVGOptimizer.optimize.then (C:\Users\NOPE\myproject\node_modules\svg-spritemap-webpack-plugin\lib\index.js:236:32)
    at process._tickCallback (internal/process/next_tick.js:68:7)
(node:84124) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:84124) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by cascornelissen** [comment]
Thanks for the issue, I'll start looking into this! For now please downgrade to `3.0.0`.

**Comment by cascornelissen** [comment]
@ZebraFlesh, this should be fixed in [`3.0.2`](https://github.com/cascornelissen/svg-spritemap-webpack-plugin/releases/tag/v3.0.2), thanks again for reporting!

**Comment by ZebraFlesh** [comment]
Thanks for the quick turn around!
